src/components/Filters/components/DateRangePicker/DateRangePicker.tsx

```diff
@@ -35,6 +35,7 @@ const DateRangePicker: React.FC<DateRangePickerProps> = ({
       <DatePicker
         calendarClassName="DateRangePicker-calendar"
         className="DateRangePicker-input"
+        dateFormat="d MMM, yyyy"
```

Comment:
So what are the valid input formats? Can I use different formats or just this one?

Reply:
`d MMM, yyyy` is display format and you can input any of these: `MM/DD/YYYY, YYYY/MM/DD, feb 2021, 12 feb 2021, 2021 feb 12, 2021 12 feb`. Parsing itself is done by library so it is much less work than I thought. Even more, we found out Atlas is using same library and ours is most likely first date picker in platform. The refactoring paid off in the end.